src/styles/calendar.scss

```
@@ -0,0 +1,42 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding * 2 $padding;
```

it will make the rule more readable
```suggestion
  padding: (2 * $padding) $padding;
```